spyder/plugins/pythonpath/utils.py

```diff
@@ -17,14 +17,20 @@
 
 def check_path(path):
     """Check that `path` is not a [site|dist]-packages folder."""
+    pattern_string = r'(ana|Ana|mini|micro)(conda|mamba|forge)\d*(/base)*/pkgs'
```

Comment:
Does this pattern take into account capitalized Conda distribution names such as `Miniconda` or `Mambaforge`? I think those names are the default on Windows, so we need to cover them too.